packages/client/src/StreamrClient.ts

```diff
@@ -554,14 +555,27 @@ export class StreamrClient {
         return this.node.getNode()
     }
 
+    getEntryPoints(): PeerDescriptor[] {
```

Comment:
How this method is used in practice? We set the entry point separately for each stream/streamPart, but here we don't have anything about streams?

Reply:
Removed this function as it is no longer required.